pngcrush.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: pngcrush
+  version: 1.8.13
+  epoch: 0
+  description: PNG recompression utility
+  copyright:
+    - license: Zlib
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libpng-dev
+      - zlib-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 3b4eac8c5c69fe0894ad63534acedf6375b420f7038f7fc003346dd352618350
+      uri: https://downloads.sourceforge.net/pmt/pngcrush-${{package.version}}-nolib.tar.xz
+
+  - uses: patch
+    with:
+      patches: alder32.patch
```

Comment:
Could you provide context for why we vendor this patch?

Reply:
without this patch, we're running into build failure issues. I tried to pickup a patch upstream for this but wasn't able to find one. So commented out the part.

Reply:
Nit, but could we vendor Alpine's patch instead?: https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/community/pngcrush